src/TagLib/Aac/File.cs

```diff
@@ -219,7 +219,7 @@ public override TagLib.Tag GetTag (TagTypes type, bool create)
 		{
 			// Only check the first 16 bytes so we're not stuck
 			// reading a bad file forever.
-			if (propertiesStyle != ReadStyle.None &&
+			if (propertiesStyle.HasFlag(ReadStyle.Average) &&
```

Comment:
I think `.HasFlag` is terrible for performance. It is probably better to avoid `.HasFlag` usage in general.

Reply:
True. I wasn't aware of this cost penalty. However, these constructs are used in non-critical paths (non-recursive, outside the loops). In my view, the cost penalty is insignificant, and the readability gain is welcome in this kind of one-time-path.

Still, I can accept that there is a coding-style rule for TagLib# like: "Never use `HasFlag`". So I'll remove every occurrence of it in the code.